### ISSUE TITLE
aria-ng-gui@2.0.1: download via aria2 failed

### ISSUE THREAD

**Issue by Volipy** [lukesampson/scoop-extras#2911]
ERROR Download failed! (Error 22) HTTP response header was bad or unexpected
ERROR https://github.com/Xmader/aria-ng-gui/releases/download/v2.0.1/AriaNgGUI-win32-x64.zip
    referer=https://github.com/Xmader/aria-ng-gui/releases/download/v2.0.1/
    dir=C:\Users\10027\scoop\cache
    out=aria-ng-gui#2.0.1#https_github.com_Xmader_aria-ng-gui_releases_download_v2.0.1_AriaNgGUI-win32-x64.zip

ERROR & 'C:\Users\10027\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\10027\scoop\cache\aria-ng-gui.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=13552 --continue

**Comment by Ash258** [comment]
Supressed by eaa70c55c42df0d9f19bf76909811621ee0c5b92
